src/Configuration/ConfigurationFactory.php

```diff
@@ -102,7 +102,7 @@ public function create(
         bool $noProgress,
         ?bool $ignoreMsiWithNoMutations,
         ?float $minMsi,
-        bool $showMutations,
+        ?int $showMutations,
```

Comment:
minor, but to be consistent with other files:

```suggestion
        ?int $numberOfShownMutations,
```

and method will be renamed as well.

Here and in a couple of other places